### PR TITLE
default policy management is now optional

### PR DESCRIPTION
**Pull request by bovy89**

### CODE REVIEW

**Comment by jyaworski** [comment]
What is the use-case for this?

**Comment by bovy89** [comment]
I don't want to manage default admin/api policy with an hash conf. I need to manage it with a custom erb template

**Comment by jyaworski** [comment]
It looks like we have a couple of undocumented parameters.

```puppet
 $api_template = $rundeck::params::api_template,
 $acl_template = $rundeck::params::acl_template,
```

Does this suit your needs?

**Comment by bovy89** [comment]
These parameters seem unused. Template file path is hardcoded
`content => template("${module_name}/aclpolicy.erb"),`

**Comment by jyaworski** [comment]
Huh. Could this PR be to actually make those do something, then?

Good catch.

File: manifests/config.pp [hunk 1]
```diff
@@ -119,18 +121,22 @@
     require => File[$properties_dir],
   }
 
-  rundeck::config::aclpolicyfile { 'admin':
-    acl_policies   => $acl_policies,
-    owner          => $user,
-    group          => $group,
-    properties_dir => $properties_dir,
-  }
+  if($manage_default_policy == true){
```
Comment (jyaworski) [inline manifests/config.pp]: This could just be:

```puppet
if $manage_default_policy {
```

File: manifests/init.pp [hunk 2]
```diff
@@ -159,6 +168,7 @@
   $rdeck_home                   = $rundeck::params::rdeck_home,
   $rdeck_config_template        = $rundeck::params::rdeck_config_template,
   $file_keystorage_keys         = $rundeck::params::file_keystorage_keys,
+  $manage_default_policy        = $rundeck::params::manage_default_policy,
```
Comment (jyaworski) [inline manifests/init.pp]: It looks like this is being used for both the api and acl policies. We should change this to policies, or add another variable for api.

**Comment by bovy89** [comment]
@jyaworski Can you release a new version of this module asap?

Thanks

**Comment by jyaworski** [comment]
@bovy89 why the rush?

@liamjbennett I believe we just released a new module a short time ago. Should this go in anyway?

**Comment by bovy89** [comment]
I just wanted to know if was possible a new release of this module in a couple of days; otherwhise I need to include my feature in another way

**Pull request merged**